CaloReco/fcl/prolog_trigger.fcl

```diff
@@ -12,12 +12,12 @@ BEGIN_PROLOG
 FastCaloRecoDigiMaker :
 { 
     module_type            : FastCaloRecoDigiFromDigi
-    caloDigiModuleLabel    : CaloDigiMaker
+    caloDigiCollection     : CaloDigiMaker
+    ProtonBunchTime        : "EWMProducer"
```

Comment:
which module is used to provide the ProtonBunchTime data-product? Does it need to be added in the `prepareHits` sequence?